src/tests/functional/shared_test_classes/src/single_op/search_sorted.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/search_sorted.hpp"
+
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+
+static const int SEED = 7877;
+
+std::string SearchSortedLayerTest::getTestCaseName(testing::TestParamInfo<SearchSortedLayerTestParams> obj) {
+    SearchSortedLayerTestParams basicParamsSet;
+    basicParamsSet = obj.param;
+
+    SearchSortedSpecificParams searchSortedParams;
+
+    ElementType inputPrecision;
+    std::string targetDevice;
+    std::tie(searchSortedParams, inputPrecision, targetDevice) = basicParamsSet;
```

Comment:
Better to use reference to avoid extra copy `const auto& p = std::get<n>(basicParamsSet)`

Reply:
Every single test on CPU side I have checked is doing it this way - so will keep it to be that way to be consistent.